ComponentKit/Core/Scope/CKDetectComponentScopeCollisions.mm

```diff
@@ -58,5 +119,17 @@ void CKDetectComponentScopeCollisions(const CKComponentLayout &layout)
       }
     }
   }
+  return CKComponentCollision();
+}
+
+void CKDetectComponentScopeCollisions(const CKComponentLayout &layout) {
+#if CK_ASSERTIONS_ENABLED
+  CKComponentCollision collision = CKReturnComponentScopeCollision(layout);
+  if (collision.hasCollision()) {
+    CKCFailAssert(@"Scope collision. Attempting to create duplicate scope for component: %@ with lca: %@\n%@",
```

Comment:
Let's avoid abbreviations in the error messages. Folks won't know what they mean.